src/Catalyst.Core.Modules.Dfs/CoreApi/BootstrapApi.cs

```diff
@@ -0,0 +1,130 @@
+#region LICENSE
+
+/**
+* Copyright (c) 2019 Catalyst Network
+*
+* This file is part of Catalyst.Node <https://github.com/catalyst-network/Catalyst.Node>
+*
+* Catalyst.Node is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 2 of the License, or
+* (at your option) any later version.
+*
+* Catalyst.Node is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with Catalyst.Node. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#endregion
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Catalyst.Abstractions.Dfs.CoreApi;
+using Catalyst.Abstractions.Options;
+using MultiFormats;
+using Newtonsoft.Json.Linq;
+
+namespace Catalyst.Core.Modules.Dfs.CoreApi
+{
+    public class BootstrapApi : IBootstrapApi
+    {
+        // From https://github.com/libp2p/go-libp2p-daemon/blob/master/bootstrap.go#L14
+        public static MultiAddress[] Defaults =
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use an immutable collection or reduce the accessibility of the field(s) 'Defaults'.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4832783)